bin/MadGraph5_aMCatNLO/cards/production/13TeV/TTToHcToWA_AToMuMu/TTToHcToWA_AToMuMu_MHc160_MA15/TTToHcToWA_AToMuMu_MHc160_MA15_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+import model 2HDM_UFO-type1_5FS
+
+define p = g u d c s b u~ d~ c~ s~ b~
+define j = p
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+definve vl~ = ve~ vm~ vt~
+
+generate p p > t t~, (t > b h+, (h+ > w+ h3, (h3 > mu+ mu-), (w+ > l+ vl))), (t~ > b~ w-, (w- > j j )) @0
+add process p p > t t~,(t > b h+,(h+ > w+ h3,(h3 > mu+ mu-),(w+ > j j))),(t~ > b~ w-,(w- > l- vl~)) @1
+add process p p > t t~,(t > b h+,(h+ > w+ h3,(h3 > mu+ mu-),(w+ > l+ vl))),(t~ > b~ w-,(w- > l- vl~)) @2
```

Comment:
You don't need to split the proc cards into two : 
1. (h+ > w+ h3, (h3 > mu+ mu-), (w+ > l+ vl)))
2. (h+ > l+ vl h3, (h3 > mu+ mu-))

For "m(h+) < m(w+) + m(h3)", only 2. will work.
But for "m(h+) > m(w+) + m(h3)", 1. and 2. will both work.
Why just not use 2. for all mass scenarios?

Reply:
Thanks for the comment,
I changed to use both 1. and 2. for all the mass points.